### DISCUSSION TITLE
Como editar códigos comprados no CodeCanyon?  (Novato Pede Ajuda)

### DISCUSSION THREAD

**Discussion by fmidia** [Flutterando/forum#150] [Perguntas e Respostas]
Boa tarde

Recebo muitos pedidos de clientes que compram scripts prontos no CODECANYON, e assim vou adaptando os códigos ao que o cliente precisa, porém sou totalmente NOVATO no flutter, e realmente quero aprender 100% esse sistema, 

Sempre que faço download do código flutter comprado por um cliente, surgem muitos erros grafados e sublinhados em vermelho, e realmente não faço a mínima ideia como resolver essa quantidade de erros, sei que pode ser algo sem sentido, mas estou procurando ajuda aqui, pq realmente quero ficar não dependendo de suportes na India e etc....

![3](https://github.com/Flutterando/forum/assets/70591738/2eb057de-5b18-4fc6-bbc8-565bbccf9978)

**Alguém tem ideia como fazer para deixar o código limpo, rodando sem essas marcações de erros? Deixar funcional para fazer a geração das chaves ?

No caso ai, a lógica e todos projetos já estão prontos, só quero deixar funcional, gerar o build para android e IOS e colocar em produção, aceito, qualquer tipo de ajuda!!**

**Comment by gkpiccoli** [comment]
E ai @fmidia blza? Talvez você precise dar um get nas dependencias do projeto. No projeto desejado abre o terminal e digita o seguinte: 

flutter clean

E depois:

flutter pub get 

Caso você encontre mais algum erro, utilize o:

dart fix --aply

Acho que é isso, boa sorte.

**Reply by fmidia** [reply]
Gustavo blz?
obrigado pela resposta...infelizmente eu desisti, já tentei todas as formas possíveis e não consegui, me falaram para instalar o FVM e deixar a versão do código usado. Mesmo assim não consegui....
Sou novo no flutter, e pelo jeito vou precisar de suporte externo pra realizar isso. Se vc fizer esse trabalho, me dá um toque.

abs

**Reply by gkpiccoli** [reply]
Eu tbm sou novato mas será um prazer te ajudar. Me adiciona no Discord que a gente pode se aprofundar nisso. Minha conta é no discord é: Gustavo Kureck Piccoli#7261
Lembra de marcar o tópico como resolvido, ou fechar ele. 
Abraço!

**Comment by Developer-jdantas** [comment]
Nesses casos fica mais fácil ajudar com o código na tela, são mil possibilidades que podem estar ocorrendo, é preciso passar um pente fino no código que você recebeu.
